src/services/diskLRU.ts

```diff
@@ -53,8 +53,13 @@ class DiskLRUService {
                     if (size >= maxSize) {
                         const leastRecentlyUsed =
                             await this.findLeastRecentlyUsed(cacheDir);
-
-                        await unlink(leastRecentlyUsed.path);
+                        try {
+                            await unlink(leastRecentlyUsed.path);
+                        } catch (e) {
+                            if (e.code === 'ENOENT') return;
+                            logError(e, 'Failed to evict least recently used');
```

Comment:
Should we log before the return?

Reply:
ENOENT error is basically, file not found, which we can ignore as we are anyways trying to delete the file

Reply:
Have added comments to clarify